packages/dashboard/src/components/grid/gestures/useDragMonitor.ts

```diff
@@ -33,6 +33,8 @@ export const useDragMonitor = ({
   enabled,
   dashboardGrid,
 }: DragMonitorProps) => {
+  const rafRef = useRef<number>();
```

Comment:
do you need a ref here ? i.e 
``` const anyVar = requestAnimationFrame() ```

Reply:
Yeah, we need a ref here so that we maintain the reference to the current animation frame after the re-render. Without this, we wouldn't be able to cancel the right animation frame at unmount.

Reply:
Oh, no, you're right. We can avoid it since everything can be contained in the effect. I'll change that!